### ISSUE TITLE
For the experiment.

### ISSUE THREAD

**Issue by daikiokamura** [hongxin001/JoCoR#10]
Dear Hongxin Wei,

I have three questions for the clothing 1M experiment.
First, what was the value of τ (noise rate) used for small loss selection?
Second, did you change the momentum value of the Adam optimizer to epoch like the Cifar10 dataset?
Finally, you mentioned that you used Resnet18. Did you use the pre-trained parameters?

And, one more question for the CIFAR-10 and CIFAR-100 experiments.
What did you use for the clean validation set when selecting the hyperparameter lambda?
As far as I know, CIFAR-10 and CIFAR-100 datasets contain training and test data (no validation data).

I carefully read your paper, but I couldn't find such an explanation.

Thank you.

**Comment by rayleizhu** [comment]
> Dear Hongxin Wei,
> 
> I have three questions for the clothing 1M experiment.
> First, what was the value of τ (noise rate) used for small loss selection?
> Second, did you change the momentum value of the Adam optimizer to epoch like the Cifar10 dataset?
> Finally, you mentioned that you used Resnet18. Did you use the pre-trained parameters?
> I carefully read your paper, but I couldn't find such an explanation.
> 
> Thank you.

I have the same questions, besides, I also wanna ask what's optimal co_lambda on Clothing1M. In summary:

1. What is the noise rate to set?
2. What is optimal co_lambda?
3. What is the learning rate schedule?

Looking forward to your kind reply. Thanks. @hongxin001

**Comment by hongxin001** [comment]
Thanks for your interest. For clothing-1m, the schedules for learning rate and forget rate are described in subsection 4.1 Experiment setup.

As for the noise rate, you may need to try 0.2, 0.3, 0.4 based upon the performance on the validation set.

I didn't use pre-trained weights for initialization.
